tests/phpunit/tests/blocks/wpBlock.php

```diff
@@ -170,7 +170,7 @@ public function test_constructor_assigns_context_from_block_type() {
 		);
 		$block        = new WP_Block( $parsed_block, $context, $this->registry );
 
-		$this->assertSame( array( 'requested' => 'included' ), $block->context );
+		$this->assertSame( $context, $block->context );
```

Comment:
This isn't working as intended in the branch. If the block doesn't define `unrequested` in the `uses_context`, it should not be exposed to the block.

Reply:
It because the block register with those context. That is the goal of these PR. Please correct me if i miss anything. Thanks!

Reply:
Removed in https://github.com/WordPress/wordpress-develop/pull/7522/commits/c83ca8268e3b9db13ad3e17cf0e2a478529dbdc6